### DISCUSSION TITLE
Need help to remove gnome for qtile?

### DISCUSSION THREAD

**Discussion by abdoulayegk** [qtile/qtile#3306] [Q&A]
I installed arch linux with gnome and I install qtile and make it look the way i wanted now i want to remove gnome but i am afraid of removing it will broke something. Any advice for me?

**Comment by m-col** [comment]
Qtile is only configured through the `config.py` file so anything else will be specific to that app or to gnome. gnome won't touch the Qtile config so you'll be fine there.
